tiledb/sm/filter/filter_pipeline.cc

```diff
@@ -256,7 +256,8 @@ Status FilterPipeline::filter_chunks_forward(
 }
 
 Status FilterPipeline::filter_chunks_reverse(
-    const std::vector<std::tuple<void*, uint32_t, uint32_t, uint32_t>>& chunks,
+    const std::vector<std::tuple<void*, uint32_t, uint32_t, uint32_t, bool>>&
```

Comment:
the new `bool` in this tuple is true if the decompression (aka filter pipeline reversal) should be **skipped** on the associated chunk.